app/Cache/CacheServiceProvider.php

```diff
@@ -33,5 +36,16 @@ public function register(): void
                 )
             )
         );
+
+        $container->addShared(
+            CacheInterface::class,
```

Comment:
Maybe create a factory like @steftrenson did for SAPI3 so we can easily create and update the different caches. Because now they already work with configurable prefix/namespace.